Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/ReflectionAdapter.cpp

```diff
@@ -469,43 +480,19 @@ namespace AZ::DocumentPropertyEditor
             }
             else
             {
-                AZStd::string_view labelAttribute = "MISSING_LABEL";
-                Dom::Value label = attributes.Find(Reflection::DescriptorAttributes::Label);
-                if (!label.IsNull())
-                {
-                    if (!label.IsString())
-                    {
-                        AZ_Warning("DPE", false, "Unable to read Label from property, Label was not a string");
-                    }
-                    else
-                    {
-                        labelAttribute = label.GetString();
-                    }
-                }
-
-                AZ::Reflection::AttributeDataType serializedPathAttribute =
-                    attributes.Find(AZ::Reflection::DescriptorAttributes::SerializedPath);
-
-                auto containerAttribute = attributes.Find(AZ::Reflection::DescriptorAttributes::Container);
+                auto containerAttribute = attributes.Find(Reflection::DescriptorAttributes::Container);
                 if (!containerAttribute.IsNull())
                 {
                     auto container = AZ::Dom::Utils::ValueToTypeUnsafe<AZ::SerializeContext::IDataContainer*>(containerAttribute);
                     m_containers.SetValue(m_builder.GetCurrentPath(), BoundContainer{ container, access.Get() });
-                    size_t containerSize = container->Size(access.Get());
 
-                    if (containerSize == 1)
+                    if (isLabelAvailable)
                     {
                         m_adapter->CreateLabel(
                             &m_builder,
-                            AZStd::string::format("%s (1 element)", labelAttribute.data()),
-                            serializedPathAttribute.GetString());
-                    }
-                    else
-                    {
-                        m_adapter->CreateLabel(
-                            &m_builder,
-                            AZStd::string::format("%s (%zu elements)", labelAttribute.data(), container->Size(access.Get())),
-                            serializedPathAttribute.GetString());     
+                            AZStd::string::format(
+                                "%s (%zu element(s))", labelAttribute.GetString().data(), container->Size(access.Get())),
+                            serializedPath);
```

Comment:
yeah! Just thought it could be simplified a bit but it did alter the UX.

I will take some screenshots and we can discuss which one we can go with. I'm okay with reverting this new behavior.

Reply:
As long as the labels still match what we would see when using the RPE then we're good.

The 2 cases are shown in the screenshot below (n=0 and n=>1 are both the plural case, so only 2 cases even though there are 3 things highlighted :) ).

![image](https://user-images.githubusercontent.com/104796591/223495341-36c6329b-144e-4cce-b270-8c17baf91f72.png)

Reply:
Thanks for putting up this. By looking at this, I think it would be good to just revert my changes.